.github/workflows/ci.yml

```
@@ -0,0 +1,41 @@
+name: CI
+
+on: [ push, pull_request ]
+
+jobs:
+   build:
+      runs-on: ubuntu-latest
+      steps:
+         -
+            uses: actions/checkout@v3
+            with:
+               fetch-depth: 0 # Fetch all history
+         -
+            uses: actions/setup-node@v3
+            with:
+               node-version-file: '.nvmrc'
+         - run: npm i -g npm@8.5.5
+         - run: npm ci
+         - run: npm run standards
+   test:
+      needs: [ build ]
+      runs-on: ubuntu-latest
+      strategy:
+         fail-fast: false
+         matrix:
+            node-version: [ 12, 14, 16, 'lts/*', 'latest' ]
+      steps:
+         - uses: actions/checkout@v3
+         -
+            name: Use Node.js ${{ matrix.node-version }}
+            uses: actions/setup-node@v3
+            with:
+               node-version: ${{ matrix.node-version }}
+         - run: npm i -g npm@8.5.5
+         - run: npm ci # Reinstall the dependencies to ensure they install with the current version of node
+         - run: npm test
+   coverage:
+      needs: [ build, test ]
+      runs-on: ubuntu-latest
+      steps:
+         - cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage
```

Appears there might be some issues with this: https://github.com/pbredenberg/serverless-plugin-cloudfront-lambda-edge/actions/runs/4385594954

Does this work?

```suggestion
         - run: npm test
         - run: cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage
```
 
There's also a chance we may need to switch to https://github.com/marketplace/actions/coveralls-github-action

---

Nice suggestion!

Seems to have done the job: 

- https://github.com/pbredenberg/serverless-plugin-cloudfront-lambda-edge/actions/runs/4387181255
- https://coveralls.io/github/pbredenberg/serverless-plugin-cloudfront-lambda-edge